pastas/stats/signatures.py

```diff
@@ -1792,8 +1798,21 @@ def summary(
         # Get the function and compute the signature for each column/series
         func = getattr(ps.stats.signatures, signature)
         if isinstance(data, DataFrame):
-            result.loc[signature] = data.apply(func)
+            for col in data.columns:
```

Comment:
I think it is nicer that data is always a dataframe internally. Saves some lines of code. So on line 1785:
````python
elif instance(data, Series):
   result = ...
   data = data.to_frame()
```

Reply:
Ah yes I see you point, I'lkl rewrite it like that and save some lines of code